lib/ansible/modules/cloud/azure/azure_rm_roledefinition_facts.py

```diff
@@ -58,60 +60,64 @@
-      role_name:
-        description: Role name.
-        returned: always
-        type: str
-        sample: myCustomRoleDefinition
-      name:
-        description: System assigned role name.
-        returned: always
-        type: str
-        sample: xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
-      assignable_scopes:
-        description:
-            - List of assignable scope of this definition.
-        returned: always
-        type: list
-        sample: "/subscriptions/xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx/resourceGroups/myResourceGroup"
-      permissions:
-        description:
-            - List of Role Definition peremissions.
-        returned: always
-        contains:
-            actions:
-                description:
-                    - List of allowed actions.
-                returned: always
-                type: list
-                sample: Microsoft.Compute/virtualMachines/read
-            not_actions:
-                description:
-                    - List of denied actions.
-                returned: always
-                type: list
-                sample: Microsoft.Compute/virtualMachines/write
-            data_actions:
-                description:
-                    - List of allowed data actions.
-                returned: always
-                type: list
-                sample: Microsoft.Storage/storageAccounts/blobServices/containers/blobs/read
-            not_data_actions:
-                description:
-                    - List of denied actions.
-                returned: always
-                type: list
-                sample: Microsoft.Storage/storageAccounts/blobServices/containers/blobs/write
+        id:
+            description:
+                - Role Definition ID.
+            returned: always
+            type: str
+            sample: "/subscriptions/xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx/providers/Microsoft.Authorization/roleDefinitions/xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx"
+        role_name:
+            description:
+                - Role name.
+            returned: always
+            type: str
+            sample: myCustomRoleDefinition
+        name:
+            description:
+                - System assigned role name.
+            returned: always
+            type: str
+            sample: xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
+        assignable_scopes:
+            description:
+                - List of assignable scopes of this definition.
+            returned: always
+            type: list
+            sample: [ "/subscriptions/xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx/resourceGroups/myResourceGroup" ]
+        permissions:
+            description:
+                - List of Role Definition permissions.
+            returned: always
+            contains:
+                actions:
+                    description:
+                        - List of allowed data actions.
```

Comment:
There are four parts to the permissions: `actions`, `not_actions`, `data_actions`, and `not_data_actions`. 

The description of `actions` should be just `List of allowed actions.`

```suggestion
                        - List of allowed actions.
```